homeassistant/components/netgear/switch.py

```diff
@@ -166,8 +167,7 @@ def __init__(
         """Initialize a Netgear device."""
         super().__init__(coordinator, router, device)
         self.entity_description = entity_description
-        self._name = f"{self.get_device_name()} {self.entity_description.name}"
-        self._unique_id = f"{self._mac}-{self.entity_description.key}"
+        self._attr_unique_id = f"{self._mac}-{self.entity_description.key}"
         self._attr_is_on = None
```

Comment:
This is the default btw

Reply:
You mean set `self._attr_unique_id = f"{self._mac}-{self.entity_description.key}"` in the base class `NetgearDeviceEntity`?
But I dont have access to the entity_description in the base class...

Reply:
No I mean the `self._attr_is_on`

Reply:
removed the default, thanks!